### PR TITLE
Added document listener to close foodTypes when not in focus

### PR DESCRIPTION
**Pull request by AlfredoSequeida**
- Added document listener to close foodTypes when search bar is not in focus.
- Fixed the span width of location input TouchableOpacity.

### CODE REVIEW

File: screens/index.js [hunk 1]
```diff
@@ -34,6 +34,24 @@ const Index = () => {
   const [popularRestaurants, setPopularRestaurants] = useState({ stores: [] });
 
   const [foodTypeScreen, showFoodTypeScreen] = useState(false);
+  const foodTypesRef = useRef(null);
+  const searchBarRef = useRef(null);
```
Comment (dani5612) [inline screens/index.js:35]: I tried doing something like this but if you click anywhere on the screen that is not the search bar, it freezes the screen and crashes the site

Comment (dani5612) [reply screens/index.js:35]: it was something about it not being able to handle a null reference

**Review by dani5612 — APPROVED** [review]
This works! :D Thank you

**Review by MayankTamakuwala — CHANGES_REQUESTED** [review]
It would be nice if you would just get rid of those console logs on lines 78 and 82 in index.js. Other than that, functionality works. just need to integrate the search API endpoint.

**Review by SeaWalks — APPROVED** [review]

**Review by MayankTamakuwala — APPROVED** [review]

**Pull request merged**